tests/market/test_market.py

```diff
@@ -117,37 +117,34 @@ def test_market_offer_delete_readonly(market):
 
 
 @pytest.mark.parametrize("market, offer, accept_offer", [
-    (OneSidedMarket(bc=NonBlockchainInterface(str(uuid4())), time_slot=now()),
+    (OneSidedMarket(bc=NonBlockchainInterface(str(uuid4())), time_slot=now(tz=TIME_ZONE)),
      "offer", "accept_offer"),
-    (BalancingMarket(bc=NonBlockchainInterface(str(uuid4())), time_slot=now()),
+    (BalancingMarket(bc=NonBlockchainInterface(str(uuid4())), time_slot=now(tz=TIME_ZONE)),
      "balancing_offer", "accept_offer"),
-    (SettlementMarket(bc=NonBlockchainInterface(str(uuid4())), time_slot=now()),
+    (SettlementMarket(bc=NonBlockchainInterface(str(uuid4())), time_slot=now(tz=TIME_ZONE)),
      "offer", "accept_offer")
 ])
 def test_market_trade(market, offer, accept_offer):
     e_offer = getattr(market, offer)(20, 10, "A", "A")
-    now = DateTime.now(tz=TIME_ZONE)
     trade = getattr(market, accept_offer)(offer_or_id=e_offer, buyer="B",
-                                          energy=10, time=now)
+                                          energy=10)
     assert trade
     assert trade == market.trades[0]
     assert trade.id
-    assert trade.creation_time == now
+    assert trade.creation_time > e_offer.creation_time
     assert trade.offer_bid == e_offer
     assert trade.seller == "A"
     assert trade.buyer == "B"
 
 
 def test_balancing_market_negative_offer_trade(market=BalancingMarket(
-    bc=NonBlockchainInterface(str(uuid4())), time_slot=now())):  # NOQA
+        bc=NonBlockchainInterface(str(uuid4())))):  # NOQA
```

Comment:
Why is there an extra indentation here?

Reply:
Because it is on the same indentation level as the function body and some versions of Pycharm complain AFAIR.